install/nginx.bash

```diff
@@ -21,43 +21,74 @@ source ../conf
 mkdir -p /var/log/nginx/$FQDN
 touch $configFile
 
+# Initial conf for letsencrypt to work
 cat > $configFile << EOF
 server {
+	listen 80;
 	server_name $FQDN *.$FQDN;
+	
+	root $basePath/$userName/wordpress;
+	
+	location ~* .well-known* {
+	autoindex on;
+	}
+	
+	location / {
+		rewrite ^ https://$FQDN;
+	}
+}
+
+EOF
+
+# If site is not enabled already, enable it
+ln -sf $configFile /etc/nginx/sites-enabled
+# Reload nginx so that letsencrypt can get cert
+systemctl reload nginx
+
+# Run letsencrypt to get cert
+echo "Running letsencrypt to generate certificate for site!"
+echo ""
+./letsencrypt.bash $userName $FQDN
 
+# SSL conf
+cat >> $configFile << EOF
+server {
+	listen 443;
+	server_name $FQDN;
+	ssl on;
+	
+	ssl_certificate /etc/letsencrypt/live/$FQDN/fullchain.pem;
+	ssl_certificate_key /etc/letsencrypt/live/$FQDN/privkey.pem;
+	
+	add_header Cache-Control "public";
+	
 	error_log /var/log/nginx/$FQDN/error.log;
 	access_log /var/log/nginx/$FQDN/access.log;
-
+	
```

Comment:
Fy på sig. Trycka in en massa döda mellanrum på tomma rader. Du behöver confa din editor att ta bort dött whitespace när du sparar :D